test/capturing_test.lua

```diff
@@ -140,19 +140,19 @@ end
 g.test_group_hook_failed = function()
     assert_shown(function(lu2)
         local group = lu2.group('test')
-        group.before_all = function()
+        group.before_all(function()
             write_to_io()
             error('custom-error')
-        end
+        end)
         group.test = function() end
     end)
 
     assert_shown(function(lu2)
         local group = lu2.group('test')
-        group.after_all = function()
+        group.after_all(function()
             write_to_io()
             error('custom-error')
-        end
+        end)
         group.test = function() end
     end)
 end
```

Comment:
These modifications do not belong to the source code changes. The test case passes without the modifications.

Please, remove or extract these modifications.

Reply:
Yep, thank you for research. Removed these changes.